src/annotator/sidebar.js

```diff
@@ -80,16 +80,18 @@ export default class Sidebar {
     /**
      * Channel for host-guest communication.
      *
-     * @type {Bridge<HostToGuestEvent,GuestToHostEvent>}
+     * TODO - Make this a Map<frame ID, PortRPC<...>>?
+     *
+     * @type {PortRPC<GuestToHostEvent, HostToGuestEvent>[]}
      */
-    this._guestRPC = new Bridge();
+    this._guestRPC = [];
```

Comment:
In future it will probably make sense to make this a `Map<frame ID, PortRPC>`. It's problematic to do that now however because the frame ID is determined by the guest itself and we only learn about it after the guest has connected. Hence for now it is just an array.